rstudio/rhel9-python-3.9/Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM ${BASE_IMAGE}
 ARG SECRET_DIR=/opt/app-root/src/.sec
```

Comment:
Can you update also to this file the Lib path as we do on upstream for c9s?

add this part under the comment `set User R Library path`:
```
RUN mkdir -p /opt/app-root/bin/Rpackages/4.3 && chmod -R a+w /opt/app-root/bin/Rpackages/4.3
ENV R_LIBS_USER /opt/app-root/bin/Rpackages/4.3
```